src/login/user-runtime-dir.c

```diff
@@ -92,39 +98,58 @@ static int user_mkdir_runtime_path(
                          uid, gid, runtime_dir_size, runtime_dir_inodes,
                          mac_smack_use() ? ",smackfsroot=*" : "");
 
+                _cleanup_free_ char *d = strdup(runtime_path);
```

Comment:
Not a big matter, but I'd like to split out this kind of cleanups into another commit.